src/week1.org

```diff
@@ -0,0 +1,124 @@
+#+TITLE: Week 1
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org
+
+* What are we doing today?
+- Trainer introductions
+- Overview of the RoboJackets software training program
+- Navigating and manipulating files and directories in the Linux terminal
+- Using Git and GitHub for version control
+
+* Who am I? - Kristaps
```

Comment:
I ended up adjusting the presentation size and margins a bit in 2f6867dbbd6c57da49a2c95765acabd3b062b007, that seemed to work out okay.